src/inverseRobot.js

```diff
@@ -24,7 +24,17 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const inverse = {};
+
+  for (const key in robot) {
+    if (!inverse[robot[key]]) {
+      inverse[robot[key]] = key;
+    } else {
```

Comment:
I think it would be better to use 'hasOwnProperty', so as not to use 'else' later, but it is only my opinion.